src/common/components/Inputs/InputNumberFormat.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Input, { TInput } from './Input'
+import NumberFormat from 'react-number-format'
+
+type Override<T1, T2> = Omit<T1, keyof T2> & T2
```

Comment:
nit: nice helper! I think it may be useful for other components as well, would be good to have a file with such helpers, in common/types.ts for example

Reply:
I added types.ts in common/utils